packages/styles/components/_c.hero.scss

```diff
@@ -68,7 +68,7 @@
   }
 
   &__content {
-    color: $color-grey-000;
+    color: $color-font-lightest;
```

Comment:
Replace -lightest to -revert

Reply:
@caqueste as seen together, this point will be fixed in a new PR on the `@next` version